rocks-db/src/dump_client.rs

```diff
@@ -199,6 +200,25 @@ impl Storage {
             synchronizer_metrics.clone(),
         ));
 
+        let mut collections = HashMap::new();
```

Comment:
we need only the core collections, this way you may save 99.99% of the memory for this map

Reply:
so the suggested approach is 
```
let pubkeys = self.asset_static_data.iter_start().filter_map(|a|a.specification_asset_class == MplCoreCollection? Some(a.pubkey):none).collect;
let collections = self.asset_collection_data
            .batch_get(pubkeys)
```